packages/snaps-utils/src/snaps.ts

```diff
@@ -422,3 +383,5 @@ export function verifyRequestedSnapPermissions(
     `The requested permissions do not have a valid ${SnapCaveatType.SnapIds} caveat.`,
   );
 }
+
+export type { Snap as TruncatedSnap } from '@metamask/snaps-sdk';
```

Comment:
Curious as to why we are doing this

Reply:
To avoid having to update this everywhere. The `Truncated` part is an implementation detail, so I've removed that from the public API. A bunch of stuff imports `TruncatedSnap` from `snaps-utils` though, so to avoid even more refactors, it's re-exported here for now.